src/both/DeltaNeutralLpV3.sol

```diff
@@ -163,6 +173,9 @@ contract DeltaNeutralLpV3 is AccessStrategy {
     /// @notice Gives ratio of vault asset to borrow asset, e.g. WMATIC/USD (assuming usdc = usd)
     AggregatorV3Interface immutable borrowAssetFeed;
 
+    address public constant WETH = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
```

Comment:
No need for this since we already have `borrowAsset`

Reply:
Removed in the new version.